src/common/snippets/include/snippets/op/staticpower.hpp

```diff
@@ -17,26 +17,28 @@ namespace op {
  * @brief Generated by Canonicalization for a spasical case of power innstruction which has constant power value
  * @ingroup snippets
  */
-class PowerStatic : public ngraph::op::v1::Power {
+class PowerStatic : public ov::op::util::UnaryElementwiseArithmetic {
```

Comment:
I think name of this file is strange. Maybe "powerstatic" would be better because op has name `PowerStatic`

Reply:
Agree, renamed